client/app/scripts/liveblog-edit/components/itemEmbedFacebook.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/no-multi-comp */
+import React, { CSSProperties, useEffect, useState } from 'react';
+import { PlaceholderEmbed } from 'react-social-media-embed';
+import { IItemMeta } from './itemEmbedInfo';
+
+declare global {
+    interface Window {
+        FB: any;
+    }
+}
+
+interface IProps extends IItemMeta {
+    style?: CSSProperties | undefined;
+}
+
+const loadFacebookLib = (callback: () => void) => {
+    const tag = 'script';
+    const id = 'facebook-js';
+    const firstScript = document.getElementsByTagName(tag)[0];
+
+    if (document.getElementById(id)) {
+        callback();
+        return;
+    }
+
+    const scriptElem = document.createElement(tag);
+
+    scriptElem.id = id;
+    scriptElem.src = 'https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v12.0';
+    scriptElem.onload = () => callback();
+
+    firstScript.parentNode.insertBefore(scriptElem, firstScript);
+};
+
+const RenderPlaceHolder = ({ url }: { url: string }) => {
+    return (
+        <PlaceholderEmbed
+            linkText="View post on Facebook"
+            url={url}
+            style={{ maxWidth: 550, height: 600 }}
+        />
+    );
+};
+
+export const ItemEmbedFacebook: React.FunctionComponent<IProps> = (props) => {
+    const [isLoading, setLoading] = useState(true);
+
+    useEffect(() => {
+        loadFacebookLib(() => {
+            setLoading(false);
+        });
+    }, []);
+
+    useEffect(() => {
+        if (!isLoading && (window as any)?.FB) {
```

Comment:
with the declaration above you don't need the casting as `any` here.
```suggestion
        if (!isLoading && window?.FB) {
```